addons/web/static/src/js/model.js

```diff
@@ -264,21 +259,20 @@ odoo.define("web/static/src/js/model.js", function (require) {
         /**
          * Method used internally to instantiate all extensions. Can also be
          * called externally to add extensions after model instantiation.
-         * @param {string} extensionName
+         * @param {string} registryName
          * @param {Object} extensionConfig
          */
-        addExtension(extensionName, extensionConfig) {
+        addExtension(registryName, extensionConfig) {
             const { name, registry } = this.constructor;
-            const Extension = registry.get(extensionName);
+            const Extension = registry.get(registryName);
```

Comment:
don't like: seems to be the name of the registry itself

Reply:
Done